GeneralSQL/Error.py,sha256=kBrYZ17K0Jps9-DY3QFG_tLa9GOrLBJvoTLv8wwxfMk,2321
GeneralSQL/__init__.py,sha256=505ZB_B0mzXGkXoWBzxo3aKaST4W-J72snG9xi11O7k,2212
GeneralSQL/Template/__init__.py,sha256=aMLYhiVXIFE5Hw9qlL10VX5caadJcWZHXUiZh3H1FIM,1435
GeneralSQL/Template/base.py,sha256=eQps_95SA2x7mI-LAeiRgFozkQ2TeOW1k6v2vIqOsHs,3023
GeneralSQL/db/Jmysql.py,sha256=aHaDjSWTVOHyNcnmeb_GNvtRdylDFN3BIBP7eyVJWXw,14674
GeneralSQL/db/__init__.py,sha256=PgWm-6m5ZDVt2vjNOtrgO7sQYzXa6A0p8Y2RnoZNTl0,57
GeneralSQL-1.0.5.dist-info/LICENSE,sha256=OXLcl0T2SZ8Pmy2_dmlvKuetivmyPd5m1q-Gyd-zaYY,35149
GeneralSQL-1.0.5.dist-info/METADATA,sha256=Jk74ACdeY-u-G8uPuMfPruz1_Kv8jHgWi2tgXF_WtY8,3487
GeneralSQL-1.0.5.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
GeneralSQL-1.0.5.dist-info/top_level.txt,sha256=N28y8Tp5KNt4vFIdlg6UXzrY9QGtXwgK-oqRvdRBuh8,11
GeneralSQL-1.0.5.dist-info/RECORD,,
